etcdmain/config.go

```diff
@@ -450,9 +513,9 @@ func initialClusterFromName(name string) string {
 	return fmt.Sprintf("%s=http://localhost:2380", n)
 }
 
-func (cfg config) isNewCluster() bool          { return cfg.clusterState.String() == clusterStateFlagNew }
+func (cfg Config) isNewCluster() bool          { return cfg.ClusterState == clusterStateFlagNew }
 func (cfg config) isProxy() bool               { return cfg.proxy.String() != proxyFlagOff }
```

Comment:
still tearing this up; proxy config stuff will be private to `etcdmain`, regular etcd config stuff will be in `embed`